example/integration_test/tests.gradle

```diff
@@ -9,7 +9,7 @@
 
 // Configure these based on your system's emulator/simulator names.
 def android_device = "Android SDK built for x86"
-def iOS_device = "iPhone 13"
+def iOS_device = "iPhone 14"
```

Comment:
Seems like I forgot to add some other files. They will be added in a later commit.